CONVENTIONS.md

```diff
@@ -116,12 +116,85 @@ Name screen coordinates `x` and `y`, rather than `left` and `top`.
 
 ## Javadocs
 
-Write sentences for class, method and fields javadocs, starting with an uppercase and ending with a period. Start method docs with verbs, like `Gets` or `Called`. Use HTML tags such as `<p>` if the docs have several paragraphs.
+Write sentences for class, method and fields javadocs, starting with an uppercase and ending with a period. Start method docs with verbs, like `Gets` or `Called`. Use HTML tags such as `<p>` if the docs have several paragraphs, as line wraps are converted to spaces in the generated documentation. Feel free to start a new line whenever you feel the current line is too long.
 
-Write quick descriptions for parameter javadocs as well as `@return` tags, with no uppercase or period. Add parameter docs to the parameter itself instead of using the `@param` tag.
+Parameter and `@return` documentation should use quick descriptions without initial capitalization or punctuation, such as `whether the block placement was successful`.
+
+Avoid using the `@param` tag for documentation of owner methods. Add parameter docs to the parameter itself instead of adding `@param` tags to its owner method. However, you can use the tag for type parameters (such as `<T>` in `public class Lazy<T>`), who don't have mapping entries for their documentations.
+
+Javadoc will take the first sentence, ended by the first `.`, as a brief description of the member you are documenting. If you need to enclose another sentence to serve as a description, such as when you have multiple periods within a sentence, enclose the summary within the `{@summary }` inline tag. An example sentence with multiple periods enclosed within a summary tag would be `{@summary Returns the client's player, i.e. the controlled player.}`.
+
+### Tooling
+
+Fabric-hosted Javadocs are generated using [JDK 11 Standard Doclet](https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html) and can use any feature it supports, though you can personally build the documentation with a newer Java version. See [the 'Checking Javadoc' section](#checking-javadoc) for how to build the documentation locally.
```

Comment:
```suggestion
Fabric-hosted Javadocs are generated using [the JDK 11 Standard Doclet](https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html) and can use any feature it supports, though you can personally build the documentation with a newer Java version. See [the 'Checking Javadoc' section](#checking-javadoc) for how to build the documentation locally.
```

Reply:
Nah, you don't always call an apple tree "the apple tree"